### PR TITLE
Show hide cart

### PR DESCRIPTION
**Pull request by SabaMallah**

### CODE REVIEW

**Comment by SabaMallah** [comment]
> _ No description provided. _

Yes it is @imambux .

File: app.js [hunk 1]
```diff
@@ -1,27 +1,76 @@
...
-            <button class="bag-btn" data-id="1">
-                <i class="fas fa-shopping-cart"></i>
-                add to bag
-            </button>
-        </div>
-        <h3>${products[index].name}</h3>
-        <h4>$${products[index].price}</h4>
-    </article>
-`;
+const getProducts = async function () {
```
Comment (imambux) [inline app.js]: why do you need this code? You just need to focus on showing and hiding cart by clicking on the icon.

**Review by imambux — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (SabaMallah) [reply app.js]: Thanks @imambux for your honest review , i have fixed the concerns.

File: app.js [hunk 2]
```diff
@@ -1,27 +1,19 @@
...
+//Showing Cart
+cartDOM.addEventListener("click", showCart);
+function showCart() {
+  cartDOM = cart.style.transform = "translateX(0%)";
+  cartDOM = cartOverlay.style.visibility = "visible";
 }
 
-productsCenterDOM.innerHTML = results;
+//CLOSE CART
+
```
Comment (imambux) [inline app.js:14]: please remove this extra line

File: app.js [hunk 3]
```diff
@@ -1,27 +1,19 @@
...
-            </button>
-        </div>
-        <h3>${products[index].name}</h3>
-        <h4>$${products[index].price}</h4>
-    </article>
-`;
+//Showing Cart
+cartDOM.addEventListener("click", showCart);
+function showCart() {
+  cartDOM = cart.style.transform = "translateX(0%)";
```
Comment (imambux) [inline app.js:9]: @SabaMallah do you need cartDOM? I think cart.style.transform = "..." will do that job and nothing is return from there, so you actually don't need to have cartDOM in this and the function below. correct me if I am wrong.

**Review by imambux — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (SabaMallah) [reply app.js:14]: okay , i am working on it.

**Pull request closed without merging**